x-pack/plugins/fleet/server/services/agent_policies/package_policies_to_agent_inputs.ts

```diff
@@ -64,6 +66,10 @@ export const storedPackagePolicyToAgentInputs = (
       ...getFullInputStreams(input),
     };
 
+    if (addFields) {
+      fullInput.processors = [addFields];
```

Comment:
could you add verification steps to the pr description, how can we verify that the processors added to the agent policy work as expected on the agent side?

Reply:
Yes absolutely. I did verify manually by hand, so I'll write down the steps that I took.